examples/apps/contact-list/package.json

```diff
@@ -15,16 +15,16 @@
   },
   "dependencies": {
     "jitar": "*",
-    "mongodb": "^6.4.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "mongodb": "*",
```

Comment:
Using a `*` as version forces us to always test this example because we loose control over incoming breaking changes. As discussed, we don't want this.